components/Icon/index.tsx

```diff
@@ -0,0 +1,67 @@
+        <path fillRule="evenodd" clipRule="evenodd" d="M7.38653 9.16575L9.43087 7.12141C9.45371 7.0987 9.46764 7.06733 9.46764 7.0327C9.46764 6.96344 9.41155 6.90723 9.34216 6.90723C9.30766 6.90723 9.27629 6.92128 9.25345 6.94399L7.20911 8.98833L5.16478 6.94399C5.14206 6.92128 5.1107 6.90723 5.07607 6.90723C5.00668 6.90723 4.95059 6.96344 4.95059 7.0327C4.95059 7.06733 4.96452 7.0987 4.98736 7.12154L7.03169 9.16575L4.98736 11.2101C4.96452 11.2328 4.95059 11.2642 4.95059 11.2988C4.95059 11.3682 5.00668 11.4243 5.07607 11.4243C5.1107 11.4243 5.14206 11.4102 5.16478 11.3875L7.20911 9.34317L9.25345 11.3875C9.27629 11.4102 9.30766 11.4243 9.34216 11.4243C9.41155 11.4243 9.46764 11.3682 9.46764 11.2988C9.46764 11.2642 9.45371 11.2328 9.43087 11.2101L7.38653 9.16575Z"/>
+        </g>
+        
+    ),
+    viewBox: '0 0 15 15',
+  },
+  calendar: {
+    path: (
+      <g>
+        <path fillRule="evenodd" clipRule="evenodd" d="M4.07168 0.945312C4.28827 0.945312 4.46386 1.1209 4.46386 1.33749V1.72967H6.81694V1.33749C6.81694 1.1209 6.99256 0.945312 7.20912 0.945312C7.42569 0.945312 7.60131 1.1209 7.60131 1.33749V1.72967H9.95439V1.33749C9.95439 1.1209 10.13 0.945312 10.3466 0.945312C10.5631 0.945312 10.7388 1.1209 10.7388 1.33749V1.72967H11.1309C12.6471 1.72967 13.8762 2.95877 13.8762 4.47494V11.5342C13.8762 13.0504 12.6471 14.2795 11.1309 14.2795H3.28732C1.77115 14.2795 0.542053 13.0504 0.542053 11.5342V4.47494C0.542053 2.95877 1.77115 1.72967 3.28732 1.72967H3.6795V1.33749C3.6795 1.1209 3.85509 0.945312 4.07168 0.945312ZM4.46386 2.51404H6.81694V2.90622C6.81694 3.12281 6.99256 3.2984 7.20912 3.2984C7.42569 3.2984 7.60131 3.12281 7.60131 2.90622V2.51404H9.95439V2.90622C9.95439 3.12281 10.13 3.2984 10.3466 3.2984C10.5631 3.2984 10.7388 3.12281 10.7388 2.90622V2.51404H11.1309C12.2139 2.51404 13.0918 3.39196 13.0918 4.47494V5.2593H1.32641V4.47494C1.32641 3.39196 2.20434 2.51404 3.28732 2.51404H3.6795V2.90622C3.6795 3.12281 3.85509 3.2984 4.07168 3.2984C4.28827 3.2984 4.46386 3.12281 4.46386 2.90622V2.51404ZM1.32641 6.04366V11.5342C1.32641 12.6172 2.20434 13.4951 3.28732 13.4951H11.1309C12.2139 13.4951 13.0918 12.6172 13.0918 11.5342V6.04366H1.32641Z"  strokeWidth="0.6"/>
+
+      </g>
+    ),
+    viewBox: '0 0 15 15',
+  },
+};
+
+export default function Icon({
```

Comment:
@rafinskipg this component can be imported from the `dai-ui-icons` package directly, any reason you needed to export it here?

For ex: https://github.com/makerdao/governance-portal-v2/blob/da2e66d0397ce09553fe1ce284674bb2b6c2e690/components/Header.tsx#L5

Reply:
hey @b-pmcg yes, my plan was to migrate the icons i am adding here to dai-ui when we have permissions to release new versions. This Icon component is a direct copy of the code in dai-ui

Is there any way to add new icons to dai-ui right now?